features/promotion/managing_catalog_promotions/creating_catalog_promotion.feature

```diff
@@ -132,6 +132,25 @@ Feature: Creating a catalog promotion
         And it should have "winter_sale" code and "Winter sale" name
         And it should have priority equal to 10
 
+    @api @ui @javascript
+    Scenario: Creating a new catalog promotion with priorities with others already existing
+        Given there is a catalog promotion "PHP stuff promotion" with priority 100 that reduces price by "30%" and applies on "PHP T-Shirt" variant
+        Given there is a catalog promotion "T-Shirt sale" with priority 200 that reduces price by "10%" and applies on "PHP T-Shirt" variant
+        When I want to create a new catalog promotion
+        And I specify its code as "winter_sale"
+        And I name it "Winter sale"
+        And I specify its label as "Winter sale" in "English (United States)"
+        And I describe it as "This promotion gives a 5$ discount on all products" in "English (United States)"
```

Comment:
These steps are not needed for this scenario

Reply:
without these steps label doesnt show in `And the visitor should see that the "Kotlin T-Shirt" variant is discounted from "$40.00" to "$35.00" with "Winter sale" promotion
`